cypress/integration/console/organizations/create.spec.js

```diff
@@ -0,0 +1,86 @@
+// Copyright © 2021 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+describe('Organization create', () => {
+  const organizationId = 'test-organization'
+  const userId = 'create-organization-test-user'
+  const user = {
+    ids: { user_id: userId },
+    primary_email_address: 'edit-organization-test-user@example.com',
+    password: 'ABCDefg123!',
+    password_confirm: 'ABCDefg123!',
+  }
+
+  before(() => {
+    cy.dropAndSeedDatabase()
+    cy.createUser(user)
+  })
+
+  beforeEach(() => {
+    cy.loginConsole({ user_id: user.ids.user_id, password: user.password })
+  })
+
+  it('displays UI elements in place', () => {
+    cy.visit(`${Cypress.config('consoleRootPath')}/organizations/add`)
+
+    cy.findByText('Add organization', { selector: 'h1' }).should('be.visible')
+    cy.findByLabelText('Organization ID')
+      .should('be.visible')
+      .and('have.attr', 'placeholder')
+      .and('eq', 'my-new-organization')
```

Comment:
Let's remove these checks. I know we do them for other entities too (and they should be updated as well), but I think they're not relevant enough and likely to change.